### DISCUSSION TITLE
Ban user agent syntax?

### DISCUSSION THREAD

**Discussion by Catskill909** [AzuraCast/AzuraCast#5138] [Q&A] [answered]
Super happy to see this new function!
What exactly is the syntax to use?

This is the culprit:
axios/0.21.4
Unknown Browser 0.00, Unknown OS

**Comment by Catskill909** [comment]
ok.. the following seems to be working for Banned User agent entry
axios/0.21.4
would wildcard syntax be
axios/*

**Reply by a11ymit** [reply]
Still not working for me, can you confirm this works?

**Comment by Catskill909** [comment, marked as answer]
Yes. Works for me. 
I'm unsure what the wildcard syntax is but i was only being hit by "axios/0.21.4"
I added this as a single line and restarted the station and autoDJ

**Reply by Catskill909** [reply]
FYI.
The same client at the same location started hitting my stations again
axios/0.26.0
I started using a wildcard like this and so far seems to be working
axios/*

On a sidenote. It took a couple of reboots of both the server and liquid soap to get rid of them
